src/bucket/BucketIndexImpl.h

```diff
@@ -33,13 +33,15 @@ template <class IndexT> class BucketIndexImpl : public BucketIndex
         std::streamoff pageSize{};
         std::unique_ptr<BinaryFuseFilter16> filter{};
         std::map<Asset, std::vector<PoolID>> assetToPoolID{};
+        BucketEntryCounters counters{};
 
         template <class Archive>
         void
         save(Archive& ar) const
         {
             auto version = BUCKET_INDEX_VERSION;
-            ar(version, pageSize, assetToPoolID, keysToOffset, filter);
+            ar(version, pageSize, assetToPoolID, keysToOffset, filter,
+               counters);
```

Comment:
Counters must also be added to the `load` functions as well. It might be helpful to add metric tests to the "serialize bucket indexes" test to catch issues like this. I'd also recommend running this on testnet with a few restarts to make sure the metrics work as expected. There's a command flag you can pass to dump a file with all the metrics.